docs/world api.md

```diff
@@ -824,20 +726,19 @@ def generate_output(self, output_directory: str) -> None:
 ### Slot Data
 
 If the game client needs to know information about the generated seed, a preferred method of transferring the data
-is through the slot data. This can be filled from the `fill_slot_data` method of your world by returning a `Dict[str, Any]`,
-but should be limited to data that is absolutely necessary to not waste resources. Slot data is sent to your client once
-it has successfully [connected](network%20protocol.md#connected).
-If you need to know information about locations in your world, instead
-of propagating the slot data, it is preferable to use [LocationScouts](network%20protocol.md#locationscouts) since that
-data already exists on the server. The most common usage of slot data is to send option results that the client needs
-to be aware of.
+is through the slot data. This can be filled from the `fill_slot_data` method of your world by returning
```

Comment:
"from" is a bit of a weird preposition, could also use "through" or "using" instead of "with"
```suggestion
is through the slot data. This can be filled with the `fill_slot_data` method of your world by returning
```